src/services/hasura/hasura.service.ts

```diff
@@ -265,11 +264,7 @@ export class HasuraService {
           }
         `;
 
-    try {
-      return await this.queryDb(query);
-    } catch (error) {
-      throw new HttpException('Bad request', HttpStatus.BAD_REQUEST);
-    }
+    return await this.queryDb(query);
```

Comment:
@Xitija -- why are we removing all try-catch blocks? How do we ensure to avoid unexpected error handing?